ibtk/include/ibtk/FEDataManager.h

```diff
@@ -195,9 +195,8 @@ class FEData : public SAMRAI::tbox::Serializable
     /*!
      * \brief Set the equations systems object that is associated with the
      * FEData object.
-     *
-     * @deprecated The equation systems object should be set by the constructor.
```

Comment:
Does a deprecated message still show up in the doxygen generated files?

Reply:
I thought that doxygen would pick these up but I don't see an option for it, so I'll add these back in.